src/main/java/org/edumips64/ui/swing/DineroFrontend.java

```diff
@@ -500,15 +515,29 @@ public DineroCacheOptions(char type, int level) {
 	}
 	
 	public String toString() {
-		String prefix = "-l" + level + "-" + type;
-		String cmdline = prefix + "size" + " " + size + " ";
-		cmdline += prefix + "bsize" + " " + bsize + " ";
-
-		if(assoc > 0)
-			cmdline += prefix + "assoc" + " " + assoc + " ";
-		if(ccc)
-			cmdline += prefix + "ccc" + " ";
+		// creates empty builder, capacity 16
+		StringBuilder cacheConfig = new StringBuilder();
+		// adds 9 character string at beginning
+		try {
+			String prefix = "-l" + level + "-" + type;
+			cacheConfig.append("prefix");
+
+			String cmdline = prefix + "size" + " " + size + " ";
+			cacheConfig.append("cmdline");
+
+			cmdline += prefix + "bsize" + " " + bsize + " ";
+
+			if(assoc > 0)
+				cmdline += prefix + "assoc" + " " + assoc + " ";
+			if(ccc)
+				cmdline += prefix + "ccc" + " ";
+			
+			cacheConfig.append("cmdline");
+			return cmdline;
+		}
 
-		return cmdline;
+		catch (Exception e) {
```

Comment:
Whenever you catch exceptions, at least log them so we know what went well. Ideally, capture specific sub-classes of exception and handle them in different ways, if it makes sense.

Reply:
Yes, I agree. I plan to update it similar to the ones implemented before. Thank you for pointing it out.

Reply:
Tried using printStackTrace and toString to get the trace and print the error. Let me know if I should add more details to this.